Demo/Demo/SwiftUIComponents/CardVaultViews/CardVaultView.swift

```diff
@@ -57,6 +63,55 @@ struct CardVaultView: View {
             }
         }
     }
+
+    @ViewBuilder var makeSetupTokenSection: some View {
```

Comment:
Instead of using these functions, is it more readable to instead construct these custom views directly in the CardVaultView?

Reply:
There is a limit on number of subviews and I hit it here. (I think it was like 10 or 11)